lms/djangoapps/courseware/tests/test_discussion_xblock.py

```diff
@@ -225,30 +225,32 @@ def test_studio_view(self):
         self.assertIn('data-discussion-id="{}"'.format(self.discussion_id), fragment.content)
 
     @ddt.data(
+        (True, True, True),
+        (True, True, False),
         (True, False, False),
+        (False, True, True),
         (False, True, False),
         (False, False, True),
+        (False, False, False),
```

Comment:
With these changes this DDT block becomes a little unreadable / maintainable. What's the significance of these seven cases? Should `True, False, True`, be in here, and if we are just testing all 2^3 combinations, is there a cleaner way to generate these?